Libraries/LibWeb/Bindings/WindowObject.cpp

```diff
@@ -88,6 +90,11 @@ void WindowObject::initialize()
     m_xhr_constructor = heap().allocate<XMLHttpRequestConstructor>(*this, *this);
     m_xhr_constructor->define_property("prototype", m_xhr_prototype, 0);
     add_constructor("XMLHttpRequest", m_xhr_constructor, m_xhr_prototype);
+
+    m_range_prototype = heap().allocate<RangePrototype>(*this, *this);
+    m_range_constructor = heap().allocate<RangeConstructor>(*this, *this);
+    m_range_constructor->define_property("prototype", m_range_prototype);
+    add_constructor("Range", m_range_constructor, m_range_prototype);
```

Comment:
Sorry, this needs to be updated to avoid the issue fixed in https://github.com/SerenityOS/serenity/pull/4364. You can follow the exact same pattern though :)

Also I noticed that `void WindowObject::visit_edges(Visitor& visitor)` needs to be updated to include `m_range_{constructor,prototype}`.

Reply:
No idea what that `visit_edges` method is for, but I just added two lines like the XML thing.

Reply:
It's to prevent things from being garbage collected :)